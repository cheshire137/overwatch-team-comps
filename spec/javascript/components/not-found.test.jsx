import renderer from 'react-test-renderer'

import NotFound from '../../../app/assets/javascripts/components/not-found.jsx'

describe('NotFound', () => {
  let component = null

  beforeEach(() => {
    component = <NotFound />
  })

  test('matches snapshot', () => {
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
