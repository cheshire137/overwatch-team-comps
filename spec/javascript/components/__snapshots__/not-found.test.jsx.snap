exports[`NotFound matches snapshot 1`] = `
<h1>
  404 Not Found
</h1>
`;
